bin/centreondb-smooth-backup.sh

```diff
@@ -41,20 +41,19 @@ BACKUP_DIR="/var/backup"
 SAVE_LAST_DIR="/var/lib/centreon-backup"
 SAVE_LAST_FILE="backup.last"
 DO_ARCHIVE=1
-DB_ROOT_USER="root"
-DB_ROOT_PASSWORD=""
-INIT_SCRIPT="" # try to find it
+DB_ROOT_USER="@MYSQL_REPL_USER@"
+DB_ROOT_PASSWORD="@MYSQL_REPL_PASSWD@"
```

Comment:
we should load the config file and then use the approriate variables

first load the resource file at the top of the script 

`. /etc/centreon-ha/mysql-resources.sh`

then use account:

```bash
DB_ROOT_USER="$DBREPLUSER"
DB_ROOT_PASSWORD="$DBREPLPASSWORD"
```

or just replace all the DB_ROOT_USER variables with DBREPLUSER